DiscordLabs.org/dist/presence.js

```diff
@@ -10,53 +10,57 @@ var browsingStamp = Math.floor(Date.now() / 1000);
 presence.on("UpdateData", () => {
     let presenceData = {
         largeImageKey: "logo"
-};
-    if (document.location.hostname == "bots.discordlabs.org") {
-        presenceData.details = "Viewing Page:";
-        presenceData.state = ('DiscordLabs Bot List')
-
-    if (document.location.pathname.includes("/partners")) {
-          presenceData.details = "Viewing Page:";
-          presenceData.state = ('DiscordLabs Partners')
-
-    } else if (document.location.pathname.includes("/profile/")) {
-        var priceEls = document.getElementsByClassName("uname");
-        for (var i = 0; i < priceEls.length; i++) {
-         var profilename = priceEls[i].innerText;
-            presenceData.details = "Viewing a profile:";
-            presenceData.state = profilename
-        }
-    } else if (document.location.pathname.includes("/submit/")) {
-            presenceData.details = "Viewing Page:";    
-            presenceData.state = "DiscordLabs Bot Sumbit";
-
-    } else if (document.location.pathname.includes("/submit")) {
-        presenceData.details = "Viewing Page:";  
-        presenceData.state = "DiscordLabs Bot Sumbit";
-
-    } else if (document.location.pathname.includes("/search/")) {
-        presenceData.details = `Search for:`; 
-        presenceData.state = window.location.href.slice(39).replace(/\+|%20/g, ' ')
-
-    } else if (document.location.pathname.includes("/bot/")) {
-        var priceEls = document.getElementsByClassName("botname");
-        for (var i = 0; i < priceEls.length; i++) {
-         var botname = priceEls[i].innerText;
-            presenceData.details = "Viewing a Discord bot:";
-            presenceData.state = botname
+    };
+
+    if (document.location.hostname == "status.discordlabs.org") {
+        presenceData.details = "Viewing:";
+        presenceData.state = ('Service Status')
+
+    } else if (document.location.hostname == "discordlabs.org") {
+        presenceData.details = "Viewing:";
+        presenceData.state = ('Main Page')
+
+        if (document.location.pathname.includes("/partners")) {
+            presenceData.details = "Viewing Page:";
+            presenceData.state = ('Partners')
         }
-                    }
+
+    } else if (document.location.hostname == "bots.discordlabs.org") {
+        presenceData.details = "Viewing Page:";
+        presenceData.state = ('Bot List')
+
+        if (document.location.pathname.includes("/profile/")) {
+            var priceEls = document.getElementsByClassName("uname");
+            for (var i = 0; i < priceEls.length; i++) {
+                var profilename = priceEls[i].innerText;
+                presenceData.details = "Viewing a profile:";
+                presenceData.state = profilename
+            }
+
+        } else if (document.location.pathname.includes("/submit")) {
+            presenceData.details = "Viewing Page:";
+            presenceData.state = "Bot Sumbit";
```

Comment:
Still sumbit, should be submit - regardless of a spelling mistake on the website.